common/lib/dependabot/sem_version.rb

```diff
@@ -0,0 +1,50 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "sem_version"
+require "sorbet-runtime"
+
+module Dependabot
+  class SemVersion < Dependabot::Version
```

Comment:
Not sure why we would inherit from Dependabot::Version if the intent is to make it obsolete.